contracts/ens/DCLControllerV2.sol

```diff
@@ -33,7 +33,7 @@ contract DCLControllerV2 is Ownable {
      * @param _registrar - address of the DCL registrar contract
      * @param _feeCollector - address of the fee collector
 	 */
```

Comment:
Can we add the dev notation for the new param?

Reply:
Completely forgot, nice catch